src/lib/actions/customCode.js

```diff
@@ -19,16 +19,15 @@ var loadCodeSequentially = require('./helpers/loadCodeSequentially');
 var postscribe = require('../../../node_modules/postscribe/dist/postscribe');
 var unescapeHTMLEntities = require('./helpers/unescapeHtmlCode');
 
-var extensionSettings = turbine.getExtensionSettings();
-
 var postscribeWrite = (function() {
   var write = function(source) {
     postscribe(document.body, source, {
       beforeWriteToken: function(token) {
         var tagName = token.tagName && token.tagName.toLowerCase();
+        var cspNonce = turbine.getExtensionSettings().cspNonce;
```

Comment:
Calling `turbine.getExtensionSettings().cspNonce` here means that you will call it for each token from the HTML code. I would move this call inside the module.exports function.

Reply:
@Aaronius thoughts?

Reply:
I suppose the question is whether the data element value may change between the time the action is fired and the time `beforeWriteToken` is called. I think it's probably a good idea to move it into `module.exports` and then if we find out we need to request extension settings more frequently, we can move it back into here.